### PR TITLE
update readme.md

### PR DESCRIPTION
**Pull request by imadbourouche**
fixing some mistakes
adding a nice definition of the project
adding Requirements section: for the node version =>16 especially or we will have the error: "Error: Cannot find module 'node:events' "
adding the check for applications.commands to avoid this error: "w[50001]: Missing Access"
adding some useful links for anyone don't know the guild id, client id and token of discord bot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,37 @@
 <img src="https://www.gdgalgiers.com/static/gdg_algiers-86a26a90b5a8da9cdd3127750215f051.png" alt="GDG LOGO" >
 <br>
 
-# templateJs
+# TemplateJs
 
+TemplateJs is an open source discord.js based bot template that will help you build a Discord bot in a quick and elegant way. It's Ultimate, efficient, easy-to-use and simple-to-setup Command handler for Discord bots, You just have to tweak few settings, add your commands, and you will be ready to go!
```
Comment (youben11) [inline README.md]: I'm not sure about ultimate and efficient xD

**Review by youben11 — APPROVED** [review]
LGTM! Only a small comment regarding marketing :p

File: README.md (hunk 1 above)
Comment (imadbourouche) [reply README.md]: xD, i update it

File: README.md [hunk 2]
```diff
@@ -1,33 +1,37 @@
 <img src="https://www.gdgalgiers.com/static/gdg_algiers-86a26a90b5a8da9cdd3127750215f051.png" alt="GDG LOGO" >
 <br>
 
-# templateJs
+# TemplateJs
```
Comment (akram09) [inline README.md]: I think we need to change the name of this section to "Description"

File: README.md [hunk 3]
```diff
@@ -1,33 +1,37 @@
...
 ```npm
 npm install 
   ```
-* finaly , run the bot.js file 
+* Run the bot.js file 
 ```node
 node src/bot.js 
-  ```
\ No newline at end of file
+  ```
```
Comment (akram09) [inline README.md:37]: I think we can add a section of warning of hiding the config.json file from pushing the token.

**Review by akram09 — CHANGES_REQUESTED** [review]
Thank you for the contribution @imadbourouche :pray:

File: README.md (hunk 3 above)
Comment (imadbourouche) [reply README.md:37]: Done, hope that's what you mean

**Review by abdelghanimeliani — COMMENTED** [review]
thank you

**Review by abdelghanimeliani — APPROVED** [review]

**Review by akram09 — APPROVED** [review]

**Pull request merged**